core/config/capabilities_config.go

```diff
@@ -11,7 +13,27 @@ type CapabilitiesExternalRegistry interface {
 	RelayID() types.RelayID
 }
 
+type ConnectorConfig interface {
+	NodeAddress() string
+	DonId() string
+	Gateways() []ConnectorGatewayConfig
+	// copied from wsclient.go as can't import
```

Comment:
Why not?

Reply:
It's a circular dependency in `services/gateway/network`. @DavidOrchard I think you could make this cleaner by getting rid of that circular dependency in `services/gateway/network` and then keeping the types in there.